src/proto/flwr/proto/run.proto

```diff
@@ -28,6 +28,12 @@ message Run {
   string fab_hash = 5;
 }
 
+message StatusInfo {
+  string status = 1;     // "starting", "running", "finished"
+  string sub_status = 2; // "completed", "failed", "stopped" or ""
```

Comment:
Is it possible for `sub_status` to be empty?

If not, we should update the comment:
```suggestion
  string sub_status = 2; // "completed", "failed", "stopped"
```